### PR TITLE
Extend the range of the Char Algorithm (desc.)

### PR DESCRIPTION
**Pull request by blyxyas**
Now the Char Algorithm can take every character between 32 (space bar, the first character) and 255 (space without separation), instead of every character between 64 (@) and 128 (Ç)

### CODE REVIEW

File: docs/specs.md [hunk 1]
```diff
@@ -6,23 +6,28 @@
...
-        - [Specifiable](#specifiable-1)
-        - [Non-Specifiable](#non-specifiable-1)
-- [Functions](#functions)
-    - [Utilitaries](#utilitaries)
-        - [Specifiable](#specifiable-2)
-- [Extras](#extras)
-- [Algorithms](#algorithms)
-    - [Char algorithm](#char-algorithm)
-- [Bottom notes](#bottom-notes)
+- [Braincum Language Specifications](#braincum-language-specifications)
```
Comment (qexat) [inline docs/specs.md:9]: Is it really necessary to include title and summary in the summary though?

**Review by qexat — COMMENTED** [review]
Nice PR overall, thanks for contributing

**Comment by qexat** [comment]
(sorry I have an horrible sleep schedule even if I live in France lol)

**Comment by blyxyas** [comment]
The index change was something I didn't do (at least conciously) and I just
thought it was something that Git did or some autoformatting tool did, as
soon as I get to a computer I'm going to change it.

Btw I'm Spanish so the bad sleep schedule is in my DNA

File: docs/specs.md (hunk 1 above)
Comment (blyxyas) [reply docs/specs.md]: Fixed!

**Comment by qexat** [comment]
Thanks for your answer! For now I'll keep these out of the markdown file, but if your formatter adds them automatically I might leave them then.

**Comment by qexat** [comment]
In fact, my formatter does the exact same thing ; so I'll bring them back myself.

**Pull request merged**